src/Symfony/Component/Console/Tests/Terminal/TerminalDimensionsProviderTest.php

```diff
@@ -0,0 +1,38 @@
+<?php
+
+/*
+ * This file is part of the Symfony package.
+ *
+ * (c) Fabien Potencier <fabien@symfony.com>
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+namespace Symfony\Component\Console\Tests\Terminal;
+
+use PHPUnit_Framework_TestCase;
+use Symfony\Component\Console\Terminal\TerminalDimensionsProvider;
+use Symfony\Component\Console\Terminal\TerminalDimensionsProviderInterface;
+
+class TerminalDimensionsProviderTest extends PHPUnit_Framework_TestCase
```

Comment:
The `PHPUnit_Framework_TestCase` is not imported in other Symfony tests - so it may be unified.